src/sagemaker_containers/environment.py

```diff
@@ -679,3 +689,24 @@ def model_server_workers(self):  # type: () -> int
         """Returns:
             (int): Number of worker processes the model server will use"""
         return self._model_server_workers
+
+
+@contextlib.contextmanager
+def temporary_directory(suffix='', prefix='tmp', dir=None):  # type: (str, str, str) -> None
```

Comment:
should this temporary_directory live in environment? I dont think so, this seems to be a generic context manager for temp dirs and I can see that its used at least in modules.py as well. Maybe functions.py?

Reply:
Yeah, I thought about it. My thought process was that temporary files are part of the environment. Functions namespace only include functions that deal with functions. Do you have another suggestion?